scripts/generate-chains.ts

```diff
@@ -0,0 +1,64 @@
+import fs from 'fs';
+import path from 'path';
+import chokidar from 'chokidar';
+
+const INPUT_DIR = './chains';
```

Comment:
Nit: If you specify script paths like this it will only work if you run them using yarn which always sets the CWD to the root of the project. If you would be in some subdirectory and run ts-node directly like (e.g. `ts-node ../scripts/generate-chains.ts`) the script would fail.

But since I don't see this used without `yarn` it's not a problem.

Reply:
True. I'm happy to change this to make it more flexible, but what would that look like? `path.join(__dirname, '/chains')`?

Reply:
I think that would work too, but I'd just do `path.join(__dirname, 'chains')`. The join is smart and knows what delimeter to insert (if any).